airbyte-integrations/connectors/destination-kafka/src/main/resources/spec.json

```diff
@@ -0,0 +1,223 @@
+        "type": "integer",
+        "default": 60000,
+        "order": 10
+      },
+      "enable_idempotence": {
+        "title": "Enable idempotence",
+        "description": "When set to 'true', the producer will ensure that exactly one copy of each message is written in the stream. If 'false', producer retries due to broker failures, etc., may write duplicates of the retried message in the stream.",
+        "type": "boolean",
+        "default": false,
+        "order": 11
+      },
+      "compression_type": {
+        "title": "Compression type",
+        "description": "The compression type for all data generated by the producer.",
+        "type": "string",
+        "default": "none",
+        "enum": ["none", "gzip", "snappy", "lz4", "zstd"],
+        "order": 12
+      },
+      "batch_size": {
+        "title": "Batch size",
+        "description": "The producer will attempt to batch records together into fewer requests whenever multiple records are being sent to the same partition.",
+        "type": "integer",
+        "default": 16384,
+        "order": 13
+      },
+      "linger_ms": {
+        "title": "Linger ms",
+        "description": "The producer groups together any records that arrive in between request transmissions into a single batched request.",
+        "type": "integer",
+        "default": 0,
+        "order": 14
+      },
+      "max_in_flight_requests_per_connection": {
+        "title": "Max in flight requests per connection",
+        "description": "The maximum number of unacknowledged requests the client will send on a single connection before blocking.",
+        "type": "integer",
+        "default": 5,
+        "order": 15
+      },
+      "client_dns_lookup": {
+        "title": "Client DNS lookup",
+        "description": "Controls how the client uses DNS lookups.",
+        "type": "string",
+        "default": "use_all_dns_ips",
+        "enum": [
+          "default",
+          "use_all_dns_ips",
+          "resolve_canonical_bootstrap_servers_only",
+          "use_all_dns_ips"
+        ],
+        "order": 16
+      },
+      "buffer_memory": {
+        "title": "Buffer memory",
+        "description": "The total bytes of memory the producer can use to buffer records waiting to be sent to the server.",
+        "type": "integer",
+        "default": 33554432,
+        "order": 17
+      },
+      "max_request_size": {
+        "title": "Max request size",
+        "description": "The maximum size of a request in bytes.",
+        "type": "integer",
+        "default": 1048576,
+        "order": 18
+      },
+      "retries": {
+        "title": "Retries",
+        "description": "Setting a value greater than zero will cause the client to resend any record whose send fails with a potentially transient error.",
+        "type": "integer",
+        "default": 2147483647,
+        "order": 19
+      },
+      "socket_connection_setup_timeout_ms": {
```

Comment:
is this something a user needs to configure? can we use a default?

Reply:
There is already a default value